app/controllers/users_controller.rb

```diff
@@ -1,13 +1,15 @@
 class UsersController < ApplicationController
   def new
     @user = User.new
+    redirect_root
```

Comment:
sao lại  redirect_root e

Reply:
Tương tự như ở trên nhưng với trường hợp là sign in ạ